### PR TITLE
chore: fix typo

### PR DESCRIPTION
**Pull request by li-jia-nan**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md?plain=1)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [x] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Workflow
- [ ] Other (about what?)

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English | site: add docs Use in vite |
| 🇨🇳 Chinese |          - |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before requesting a reviewing. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

---

### <samp>🤖 Generated by Copilot at 8996bfe</samp>

This pull request adds two new document files under the `docs/react` folder: `use-with-vite.en-US.md` and `use-with-vite.zh-CN.md`. These files provide a guide for using antd components in a vite project, in English and Chinese respectively. The guide helps antd users to leverage vite as a fast and modern React development tool.

### <samp>🤖 Generated by Copilot at 8996bfe</samp>

*  Add a guide for using antd components in a vite project ([link](https://github.com/ant-design/ant-design/pull/42884/files?diff=unified&w=0#diff-633679e243dc6db5eba381e38d45d2df17e5f4198e74ab6efabd68ee33a11ca3R1-R119), [link](https://github.com/ant-design/ant-design/pull/42884/files?diff=unified&w=0#diff-4bce54a391d039eaf3cfce35f84e34e0181beb209783b0ef0f1f53061a1da785R1-R119))

### CODE REVIEW

**Comment by afc163** [comment]
参考 https://github.com/ant-design/ant-design/issues/42765 把，简单点就好，现在我们的文档有点啰嗦了。

File: docs/react/use-in-typescript.en-US.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 order: 5
-title: Use in TypeScript
+title: Usage with TypeScript
```
Comment (afc163) [inline docs/react/use-in-typescript.en-US.md:3]: TS 这篇可以干掉了，直接把 create-react-app 那篇改成默认 ts 的就行。

Comment (li-jia-nan) [reply docs/react/use-in-typescript.en-US.md:3]: 好，我还想顺便加个next的，但是记得之前好像有人加过了，找不到在哪

Comment (afc163) [reply docs/react/use-in-typescript.en-US.md:3]: 没有文章，只有 repo：https://github.com/ant-design/create-next-app-antd/

Comment (afc163) [reply docs/react/use-in-typescript.en-US.md:3]: remix 也可以加，umi 那篇也可以按同一个模板简化掉

Comment (li-jia-nan) [reply docs/react/use-in-typescript.en-US.md:3]: 👌

File: docs/react/use-with-remix.zh-CN.md [hunk 2]
```diff
@@ -0,0 +1,4 @@
+---
+order: 4
+title: 在 remix 中使用
```
Comment (afc163) [inline docs/react/use-with-remix.zh-CN.md:3]:
```suggestion
title: 在 Remix 中使用
```

File: docs/react/use-in-typescript.en-US.md (hunk 1 above)
Comment (li-jia-nan) [reply docs/react/use-in-typescript.en-US.md]: 大佬你说的 umi 指的是 https://ant.design/docs/react/practical-projects-cn 这篇吗？

Comment (afc163) [reply docs/react/use-in-typescript.en-US.md]: 对

File: docs/react/use-in-typescript.zh-CN.md [hunk 3]
```diff
@@ -1,130 +0,0 @@
----
-order: 5
-title: 在 TypeScript 中使用
----
-
-使用 `create-react-app` 一步步地创建一个 TypeScript 项目，并引入 antd。
```
Comment (afc163) [inline docs/react/use-in-typescript.zh-CN.md:6]: 看看有没有其他地方有链接，记得删掉。

File: docs/react/use-in-typescript.en-US.md (hunk 1 above)
Comment (afc163) [reply docs/react/use-in-typescript.en-US.md]: 我看这里面有不少组件和 umi 的介绍，先保留吧。把文档标题改成一样的格式就行。

**Comment by li-jia-nan** [comment]
另外 antd5 在 Remix 里面有问题，Remix 的文档先不加了，后面修好了再加吧

**Review by kiner-tang — APPROVED** [review]

**Pull request merged**